apriltag/src/test/java/edu/wpi/first/apriltag/AprilTagGenerationTest.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.apriltag;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import edu.wpi.first.util.PixelFormat;
+import org.junit.jupiter.api.Test;
+
+class AprilTagGenerationTest {
+  @Test
+  void test36h11() {
+    var frame = AprilTag.generate36h11AprilTagImage(1);
+    assertEquals(PixelFormat.kGray, frame.getPixelFormat());
+    assertEquals(10, frame.getWidth());
+    assertEquals(10, frame.getHeight());
+    assertEquals(96, frame.getStride());  // default stride in apriltag library
```

Comment:
Are we good to hard code stride to 96? I assume so but just wanna make sure it can't quietly change

Reply:
It won't *quietly* change, it's hardcoded into the apriltag library.  But you're right, this test would be better handled as reading stride and using it for the following checks.